dags/synapse-datasets-created-7-days.py

```diff
@@ -163,12 +163,19 @@ def get_datasets_projects_created_7_days(**context) -> List[EntityCreated]:
     def generate_slack_message(entity_created: List[EntityCreated], **context) -> str:
         """Generate the message to be posted to the slack channel."""
         message = ":synapse: Datasets or projects created in the last 7 days \n\n"
+        public_ds_message = "Public datasets: \n\n"
+        private_ds_message = "Private datasets: \n\n"
         for index, row in enumerate(entity_created):
             if row.content_type:
                 data_type = row.content_type
             else:
                 data_type = row.node_type
-            message += f"{index+1}. <https://www.synapse.org/#!Synapse:syn{row.id}|*{row.name}*> (Type: {data_type}, Created on: {row.created_on}, Created by: <https://www.synapse.org/Profile:{row.created_by}/profile|{row.user_name_full_name}>, Public: {row.is_public})\n\n"
+            if row.is_public:
+                public_ds_message += f"{index+1}. <https://www.synapse.org/#!Synapse:syn{row.id}|*{row.name}*> (Type: {data_type}, Created on: {row.created_on}, Created by: <https://www.synapse.org/Profile:{row.created_by}/profile|{row.user_name_full_name}>, Public: {row.is_public})\n\n"
```

Comment:
The format of the message is the same for public and private. I could see formatting the message done once above the `if row.is_public` check. Then inside the `if row.is_public` check you add it to `public_ds_message` or `private_ds_message`

Reply:
Good suggestion. Thank you, Bryan!